.mergify.yml

```diff
@@ -56,7 +42,7 @@ pull_request_rules:
         add: ["auto merged"]
   - name: Automatically merge autobump PRs on CI success
     conditions:
-      - base=master
+      - base~=^(master|release-[0-9]+.[0-9]+.x)$
```

Comment:
This could be simpler `- base~=^(master|release-)` perhaps per rule beginning L19..?

Reply:
Yup, agreed.  https://docs.mergify.com/configuration/#regular-expressions